Makefile

```diff
@@ -174,8 +174,12 @@ dev.down: # Kills containers and all of their data that isn't in volumes
 dev.stop: # Stops containers so they can be restarted
 	docker-compose stop
 
+dev.provision: 
```

Comment:
If you run the provision script from your local shell, your environment won't have access to the `DEVSTACK_WORKSPACE` environment variable, defined in `options.mk`. You can export it to your environment, of course, but using this command will be easier and require fewer steps.